### ISSUE TITLE
Unexpected result for body.getOoxml

### ISSUE THREAD

**Issue by Edward-Zhou** [OfficeDev/office-js#384]
I want to get the comments in the word online by word.js, I tried with `body.getOoxml`, but it return unexpected result when I add a comment on the last words of the document.
## Expected Behavior

Ruturn all the comments when I use with `body.getOoxml`.
## Current Behavior

`body.getOoxml` did not return the comment which is added on the last words in the doc.
## Steps to Reproduce, or Live Example

* Link to live example: ______

* Additional details:
  1. Get ooxml by   

```
function xmlToCommentsProcess() {
    Word.run(function (context) {
        console.log(6000);
        return context.sync().then(function () {
            // Create a proxy object for the document body.
            var body = context.document.body;
            // Queue a commmand to get the OOXML contents of the body.
            var bodyOOXML = body.getOoxml();
            // Synchronize the document state by executing the queued commands
            // and return a promise to indicate task completion.
            return context.sync()
                .then(function () {
                    var request = {
                        ooXml: bodyOOXML.value,
                        filePath: Office.context.document.url,
                        fileName: basename(Office.context.document.url)
                    };
                    xmlToComments(request);
                });
        });
    });
}
```
  2. eg, I have a doc like  

[Doc](https://1drv.ms/u/s!AsyeWQ2z51P3h0mVNrG_TOVhXrTG)
  3. If there is no additional lines under `4567989`, the output for `bodyOOXML.value` will miss comments `645456`

  4. If I add additional lines under `4567989`, the output for `OXML.value` will contains comment `645456`

## Your Environment

* Platform [Office Online]
* Host [Word]
* Browser : Chrome

**Comment by Rick-Kirkham** [comment]
I can reproduce. Happens on Office Online, but not desktop. If there is no content at all, not even a space character after the last comment, then the comment is not included in the returned OOXML.
@JuaneloJuanelo Is this a known bug?

**Comment by JuaneloJuanelo** [comment]
sounds like a bug, tracking with 3122686

**Comment by AbidRahman-MSFT** [comment]
This issue was fixed in version 16.0.12528.1+. Please feel free to re-open the issue or create a new one if the problem still persists. Thanks.
